client/src/common/hooks/useFullscreen.js

```diff
@@ -0,0 +1,31 @@
+import { useCallback, useEffect, useState } from 'react';
+
+
+export default function useFullscreen() {
+  const [isFullScreen, setFullScreen] = useState(document?.fullscreenElement);
+
+  useEffect(() => {
+    const handleChange = () => {
+      setFullScreen(document?.fullscreenElement);
+    };
+    document.addEventListener('fullscreenchange', handleChange, false);
+    document.addEventListener('resize', handleChange, false);
+
+    return () => {
+      document.removeEventListener('fullscreenchange', handleChange);
+      document.removeEventListener('resize', handleChange);
+    };  }, []);
+
+  const toggleFullScreen = useCallback(() => {
+    if (!document.fullscreenElement) {
+      document.documentElement.requestFullscreen();
+    } else {
+      if (document.exitFullscreen) {
+        document.exitFullscreen();
+      }
+    }
+    setFullScreen(document?.fullscreenElement);
```

Comment:
I don't think we need the `?.` here. On line 20 we access this property without it.

Reply:
thank you alex. reflex from electron